modules/olm-installing-specific-version-cli.adoc

```diff
@@ -145,33 +145,41 @@ If you do not specify the Operator's catalog, running the `oc get packagemanifes
 * The catalogs contain the same Operators or Operators with the same name.
 ====
 
-. An Operator group, defined by an `OperatorGroup` object, selects target namespaces in which to generate required role-based access control (RBAC) access for all Operators in the same namespace as the Operator group.
+. If the Operator you intend to install uses the `AllNamespaces` install mode, skip this step, because the `openshift-operators` namespace already has an appropriate Operator group in place by default, called `global-operators`.
 +
-The namespace to which you subscribe the Operator must have an Operator group that matches the install mode of the Operator, either the `AllNamespaces` or `SingleNamespace` mode. If the Operator you intend to install uses the `AllNamespaces` mode, then the `openshift-operators` namespace already has an appropriate Operator group in place.
+If the Operator you intend to install uses the `SingleNamespace` install  mode and you do not already have an appropriate Operator group in place, you must create one by following these steps:
 +
-However, if the Operator uses the `SingleNamespace` mode and you do not already have an appropriate Operator group in place, you must create one:
-
-.. Create an `OperatorGroup` object YAML file, for example `operatorgroup.yaml`:
+--
+.. An Operator group, defined by an `OperatorGroup` object, selects target namespaces in which to generate required RBAC access for all Operators in the same namespace as the Operator group.
```

Comment:
I wonder if this info should go in the intro section, rather than in the steps. While more useful here, it isn't an instruction. 
Else, at the beginning of the step, before the _If the Operator_  paragraphs so that it is clear that the info applies to both.

Reply:
Yeah, this non-instructional paragraph I think was always one of the bigger problems. At first I was thinking it would be too annoying to shove into the intro, but looking again, I think I've got a way to make it work now. Will update!